### PR TITLE
Added support for Trustly & Cv Connect APM (CS2)

### PR DESCRIPTION
**Pull request by Armando-Ibarra**

### CODE REVIEW

File: src/main/java/com/checkout/payments/request/source/apm/RequestCvConnectSource.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.checkout.common.Address;
+import com.checkout.common.PaymentSourceType;
+import com.checkout.payments.request.source.AbstractRequestSource;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@EqualsAndHashCode(callSuper = true)
```
Comment (martinseco) [inline src/main/java/com/checkout/payments/request/source/apm/RequestCvConnectSource.java:11]: We should add the getter and setter annotations here

File: src/main/java/com/checkout/payments/request/source/apm/RequestTrustlySource.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.checkout.common.Address;
+import com.checkout.common.PaymentSourceType;
+import com.checkout.payments.request.source.AbstractRequestSource;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@EqualsAndHashCode(callSuper = true)
```
Comment (martinseco) [inline src/main/java/com/checkout/payments/request/source/apm/RequestTrustlySource.java:11]: Same as in RequestCvConnectSource

**Review by martinseco — APPROVED** [review]
Looks good, just two minor comments

File: src/main/java/com/checkout/payments/request/source/apm/RequestCvConnectSource.java (hunk 1 above)
Comment (Armando-Ibarra) [reply src/main/java/com/checkout/payments/request/source/apm/RequestCvConnectSource.java:15]: That's right, I forgot how to program on java :(

**Pull request merged**